pkgs/development/python-modules/django-health-check/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, sphinx
+, setuptools_scm
+, django
+, redis
+, celery
+, pytest-django
+, pytest-cov
+, pytestCheckHook
+, mock
+, gitMinimal }:
+
+buildPythonPackage rec {
+  pname = "django-health-check";
+  version = "3.16.5";
+
+  src = fetchFromGitHub {
+    owner = "KristianOellegaard";
+    repo = pname;
+    rev = version;
+    hash = "sha256-Jfzi+o4ja2sNCSPfX9eRq3WGid1gcfehhayAD1L4f2U=";
+    leaveDotGit = true;
+  };
+
+  propagatedBuildInputs = [
+    sphinx
+    setuptools_scm
+    django
+    redis
+    celery
+  ];
+
+  checkInputs = [
+    pytest-django
+    pytest-cov
```

Comment:
```suggestion
```
we don't want to generate coverage. Please remove it from the config files that add the --cov flags, too.